src/main/scala/org/allenai/pnp/semparse/EntityLinking.scala

```diff
@@ -75,4 +77,13 @@ case class Span(val start: Int, val end: Int) {
   def contains(index: Int): Boolean = {
     index >= start && index < end
   }
+}
+
+case class KnowledgeGraph(val nodePairs: List[(String, String)]) {
+  val nodeSet = nodePairs.flatMap(p => List(p._1, p._2)).toSet
```

Comment:
I'm trying to convert a list of node pairs into a set of all nodes here. That is `List((1,2), (2,3))` to `Set(1,2,3)`. `nodePairs.toSet` will still give me pairs, right?